ChangeLog.d/no-generated-files.txt

```diff
@@ -0,0 +1,7 @@
+Requirement changes
+   * If you build the development version of Mbed TLS, rather than an official
+     release, some configuration-independent files are now generated at build
+     time rather than checked into source control. This includes some library
+     source files as well as the Visual Studio solution. Perl, Python 3 and a
+     C compiler for the host platform are required. See “Generated source files
```

Comment:
From the CI: `assemble_changelog.py` doesn't like lines that are longer than 80 bytes, and doesn't seem to care that those 81 bytes are actually 79 chars.

Reply:
That annoyed me and I went and made the script work on UTF-8 instead of bytes.